<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="28" passed="28">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="108" started-at="2020-08-03T12:11:49Z" finished-at="2020-08-03T12:11:49Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="108" started-at="2020-08-03T12:11:49Z" finished-at="2020-08-03T12:11:49Z">
      <class name="com.softserve.edu.testng.Task_243_TestNG">
        <test-method status="PASS" signature="test_A_InValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_InValidData" duration-ms="9" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[-5]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task243.calculateListOfNumber(Task243.java:25)
at com.softserve.edu.testng.Task_243_TestNG.test_A_InValidData(Task_243_TestNG.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_InValidData -->
        <test-method status="PASS" signature="test_A_InValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_InValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task243.calculateListOfNumber(Task243.java:25)
at com.softserve.edu.testng.Task_243_TestNG.test_A_InValidData(Task_243_TestNG.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_InValidData -->
        <test-method status="PASS" signature="test_A_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_ValidData" duration-ms="3" started-at="2020-08-03T12:11:49Z" data-provider="validDP_A" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[com.softserve.edu.PairNaturalNumbers_DTO@3e2, com.softserve.edu.PairNaturalNumbers_DTO@400]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_ValidData -->
        <test-method status="PASS" signature="test_A_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP_A" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[50]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[com.softserve.edu.PairNaturalNumbers_DTO@3e7, com.softserve.edu.PairNaturalNumbers_DTO@461, com.softserve.edu.PairNaturalNumbers_DTO@49b]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_ValidData -->
        <test-method status="PASS" signature="test_A_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_ValidData" duration-ms="2" started-at="2020-08-03T12:11:49Z" data-provider="validDP_A" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[com.softserve.edu.PairNaturalNumbers_DTO@3e1]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_ValidData -->
        <test-method status="PASS" signature="test_A_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP_A" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_ValidData -->
        <test-method status="PASS" signature="test_A_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP_A" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[7]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_ValidData -->
        <test-method status="PASS" signature="test_A_ValidDataIntegration(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_A_ValidDataIntegration" duration-ms="3" started-at="2020-08-03T12:11:49Z" data-provider="validDP_Integration" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_A_ValidDataIntegration -->
        <test-method status="PASS" signature="test_B_InValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_InValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[-5]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task243.calculateListOfSpecNumbers(Task243.java:44)
at com.softserve.edu.testng.Task_243_TestNG.test_B_InValidData(Task_243_TestNG.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_InValidData -->
        <test-method status="PASS" signature="test_B_InValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_InValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task243.calculateListOfSpecNumbers(Task243.java:44)
at com.softserve.edu.testng.Task_243_TestNG.test_B_InValidData(Task_243_TestNG.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_InValidData -->
        <test-method status="PASS" signature="test_B_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP_B" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[com.softserve.edu.PairNaturalNumbers_DTO@400]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_ValidData -->
        <test-method status="PASS" signature="test_B_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP_B" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[50]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[com.softserve.edu.PairNaturalNumbers_DTO@461, com.softserve.edu.PairNaturalNumbers_DTO@49b]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_ValidData -->
        <test-method status="PASS" signature="test_B_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_ValidData" duration-ms="0" started-at="2020-08-03T12:11:49Z" data-provider="validDP_B" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[com.softserve.edu.PairNaturalNumbers_DTO@3e1]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_ValidData -->
        <test-method status="PASS" signature="test_B_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_ValidData" duration-ms="2" started-at="2020-08-03T12:11:49Z" data-provider="validDP_B" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_ValidData -->
        <test-method status="PASS" signature="test_B_ValidData(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP_B" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[7]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_ValidData -->
        <test-method status="PASS" signature="test_B_ValidDataIntegration(int, java.util.List)[pri:0, instance:com.softserve.edu.testng.Task_243_TestNG@396a51ab]" name="test_B_ValidDataIntegration" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP_Integration" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_B_ValidDataIntegration -->
      </class> <!-- com.softserve.edu.testng.Task_243_TestNG -->
      <class name="com.softserve.edu.testng.Task_107_TestNG">
        <test-method status="PASS" signature="InValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_107_TestNG@51081592]" name="InValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[-10]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task107.calculateMaxPower(Task107.java:17)
at com.softserve.edu.testng.Task_107_TestNG.InValidData(Task_107_TestNG.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InValidData -->
        <test-method status="PASS" signature="InValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_107_TestNG@51081592]" name="InValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task107.calculateMaxPower(Task107.java:17)
at com.softserve.edu.testng.Task_107_TestNG.InValidData(Task_107_TestNG.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InValidData -->
        <test-method status="PASS" signature="ValidData(int, int)[pri:0, instance:com.softserve.edu.testng.Task_107_TestNG@51081592]" name="ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[50]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidData -->
        <test-method status="PASS" signature="ValidData(int, int)[pri:0, instance:com.softserve.edu.testng.Task_107_TestNG@51081592]" name="ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidData -->
        <test-method status="PASS" signature="ValidData(int, int)[pri:0, instance:com.softserve.edu.testng.Task_107_TestNG@51081592]" name="ValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidData -->
        <test-method status="PASS" signature="ValidDataIntegration(int, int)[pri:0, instance:com.softserve.edu.testng.Task_107_TestNG@51081592]" name="ValidDataIntegration" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="ValidIntegrationDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidDataIntegration -->
      </class> <!-- com.softserve.edu.testng.Task_107_TestNG -->
      <class name="com.softserve.edu.testng.Task_108_TestNG">
        <test-method status="PASS" signature="testInValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_108_TestNG@7f9a81e8]" name="testInValidData" duration-ms="0" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[-5]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task108.calculateMinResult(Task108.java:16)
at com.softserve.edu.testng.Task_108_TestNG.testInValidData(Task_108_TestNG.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInValidData -->
        <test-method status="PASS" signature="testInValidData(int)[pri:0, instance:com.softserve.edu.testng.Task_108_TestNG@7f9a81e8]" name="testInValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="inValidDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
at com.softserve.edu.Task108.calculateMinResult(Task108.java:16)
at com.softserve.edu.testng.Task_108_TestNG.testInValidData(Task_108_TestNG.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInValidData -->
        <test-method status="PASS" signature="testValidData(int, [I)[pri:0, instance:com.softserve.edu.testng.Task_108_TestNG@7f9a81e8]" name="testValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[50]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[I@72b6cbcc]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidData -->
        <test-method status="PASS" signature="testValidData(int, [I)[pri:0, instance:com.softserve.edu.testng.Task_108_TestNG@7f9a81e8]" name="testValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[I@a7e666]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidData -->
        <test-method status="PASS" signature="testValidData(int, [I)[pri:0, instance:com.softserve.edu.testng.Task_108_TestNG@7f9a81e8]" name="testValidData" duration-ms="1" started-at="2020-08-03T12:11:49Z" data-provider="validDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[I@68bbe345]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidData -->
        <test-method status="PASS" signature="testValidDataIntegration(int, [I)[pri:0, instance:com.softserve.edu.testng.Task_108_TestNG@7f9a81e8]" name="testValidDataIntegration" duration-ms="2" started-at="2020-08-03T12:11:49Z" data-provider="ValidIntegrationDP" finished-at="2020-08-03T12:11:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[I@30b8a058]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidDataIntegration -->
      </class> <!-- com.softserve.edu.testng.Task_108_TestNG -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
